iphone/Classes/TiBluetoothPeripheralManagerProxy.m

```diff
@@ -67,7 +67,25 @@ - (void)startAdvertising:(id)args
     args = [args objectAtIndex:0];
     
     if ([args isKindOfClass:[NSDictionary class]]) {
-        [peripheralManager startAdvertising:args];
+        NSMutableDictionary *advertisementData = [NSMutableDictionary dictionary];
+        NSString *localNameKey;
+        NSString *serviceUUIDsKey;
+        
+        ENSURE_ARG_OR_NIL_FOR_KEY(localNameKey, args, @"localNameKey", NSString);
+        ENSURE_ARG_OR_NIL_FOR_KEY(serviceUUIDsKey, args, @"serviceUUIDsKey", NSArray);
+        
+        if (localNameKey) {
+            [advertisementData setObject:localNameKey
+                        forKey:CBAdvertisementDataLocalNameKey];
```

Comment:
The dictionary was kept flexible to allow other constant values as well, like `CBAdvertisementDataManufacturerDataKey`. This could be a breaking change then, what do you think?